content/tutorials/manage-the-rippled-server/configuration/run-rippled-as-a-validator.md

```diff
@@ -20,11 +20,15 @@ To run a `rippled` server as a validator, complete the following tasks:
 
 2. [Enable validation on your `rippled` server](#enable-validation-on-your-rippled-server).
 
-Beyond this, you can optionally complete the following tasks that help build trust in your validator:
+3. [Connect your validator to the network using public hubs](#connect-to-the-network-using-public-hubs).
+
+At this point, you are likely running your `rippled` server as an untrusted validator. It's worth noting that running an untrusted validator can also be helpful to the overall health of the network. Untrusted validators help set the standard that trusted validators are measured against. For example, if a trusted validator is disagreeing with a lot of untrusted validators, that might indicate a problem.
+
+If you want `rippled` server operators and validator list publishers to add your validator to their UNLs, making your _untrusted_ validator a _trusted_ validator, complete the following addtional tasks that help build trust in your validator:
```

Comment:
I think we should phrase this as 'how to setup a reliable and performant validator'. I think we should avoid an outlined procedure for being 'trusted'. In the future there will hopefully be many different lists, so validators should just aim to be the best they can. Being added to a list is something that happens outside of these instructions which I don't think we need to be very prescriptive about.

Reply:
Great! Thank you for this info and feedback, Daniel. Working on updating the language and structure.